### ISSUE TITLE
COMPORD Issue

### ISSUE THREAD

**Issue by OPMUSER** [OPM/opm-simulators#1930]
I'm trying to identify why wells are not producing so I'm using the ecl-strict-parsing=true parameter to see if there are any keyword issues. If I use the COMPORD keyword with this setting:

```
-- ---------- ---------- -                                                                                                                                                                                                                                               
--    WELL    COMPLETION                                                                                                                                                                                                                                                 
--    NAME       ORDER                                                                                                                                                                                                                                                   
-- ---------- ---------- -                                                                                                                                                                                                                                               
COMPORD                                                                                                                                                                                                                                                                  
   'GP05    '   'INPUT'  /                                                                                                                                                                                                                                               
/            
```                                                                                                                                                                                                                                                            
I get a message that COMPORD only DEPTH is supported. If I change the setting to:
```
-- ---------- ---------- -                                                                                                                                                                                                                                               
--    WELL    COMPLETION                                                                                                                                                                                                                                                 
--    NAME       ORDER                                                                                                                                                                                                                                                   
-- ---------- ---------- -                                                                                                                                                                                                                                               
COMPORD                                                                                                                                                                                                                                                                  
   'GP05    '   'DEPTH'  /                                                                                                                                                                                                                                               
/                                                                                                                                                                                                                                                                        
```
I get:
`UNSUPPORTED_COMPORD_TYPE: The COMPORD keyword only handles TRACK or INPUT order.`

So is this an inconsistency with ecl-strict-parsing=true or with OPM Flow?

Like I mentioned above, I'm just trying to find out why the wells are not flowing.

**Comment by OPMUSER** [comment]
Update: The inconsistency is with ecl-strict-parsing=true  - OPM Flow terminates if DEPTH is used.

**Comment by joakim-hove** [comment]
> Update: The inconsistency is with ecl-strict-parsing=true - OPM Flow terminates if DEPTH is used.

Yes - fix here: #1931

**Comment by OPMUSER** [comment]
Tusen takk
